packages/leaderboard/src/board.ts

```diff
@@ -0,0 +1,213 @@
+import { ContractKit, newKitFromWeb3, CeloContract } from '@celo/contractkit'
+import { AccountsWrapper } from '@celo/contractkit/lib/wrappers/Accounts'
+import Web3 from 'web3'
+import http from 'http'
+import fs from 'fs'
+import readline from 'readline'
+import { google } from 'googleapis'
+import { Client } from 'pg'
+import { verifyClaim, Claim } from '@celo/contractkit/lib/identity/claims/claim'
+import { ClaimTypes } from '@celo/contractkit/lib/identity'
+
+// If modifying these scopes, delete token.json.
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+// The file token.json stores the user's access and refresh tokens, and is
+// created automatically when the authorization flow completes for the first
+// time.
+const TOKEN_PATH = 'token.json'
+
+function addressToBinary(a: string) {
+  try {
+    if (a.substr(0, 2) == '0x') return a.substr(2)
+    else return a
+  } catch (_err) {
+    return a
+  }
+}
+
+async function updateDB(lst: any[][]) {
+  const client = new Client({ database: 'blockscout' })
+  await client.connect()
+  const res = await client.query(
+    'INSERT INTO competitors (address, multiplier)' +
+      " SELECT decode(m.address, 'hex') AS address, m.multiplier FROM json_populate_recordset(null::json_type, $1) AS m" +
+      ' ON CONFLICT (address) DO UPDATE SET multiplier = EXCLUDED.multiplier RETURNING *',
+    [
+      JSON.stringify(
+        lst.map((a) => {
+          return { address: addressToBinary(a[0]), multiplier: a[1] }
+        })
+      ),
+    ]
+  )
+  console.log(res.rows)
+  await client.end()
+  await readAssoc(lst.map((a) => a[0].toString()))
+}
+
+async function updateRate(kit: ContractKit) {
```

Comment:
how to test this?

Reply:
Do you mean changing the rate in the oracle? Otherwise, I have just checked the value from the db.

Reply:
ok, so the oracle service has to be running